### ISSUE TITLE
[Accessibility] "xterm-accessibility" class div does not have tabindex so that keyboard users cannot land in this area using tab key.

### ISSUE THREAD

**Issue by jooyoungseo** [microsoft/vscode#135920]
CC @isidorn

## System Info

Version: 1.62.0-insider (system setup)
Commit: dc1a6699060423b8c4d2ced736ad70195378fddf
Date: 2021-10-26T08:49:04.076Z
Electron: 13.5.1
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.19043

## Current Issue

`"xterm-accessibility"` class div does not have tabindex so that keyboard users cannot land in this area using tab key.

## Reproducible Steps

1. Load NVDA.
2. Launch VSCode.
3. Create terminal.
4. Focus in terminal input by pressing Ctrl+`.
5. Press Shift+Tab to focus in terminal output area.
6. Use up/down-arrows to see if NVDA reads any terminal lines.

* Note: I know, we can use Ctrl+Up-Down-arrows to go to previous/next line in the terminal and these will put focus in the terminal output area; however, currently, there is no way to land in the terminal output div because `"xterm-accessibility"` class div having "document" role does not have tabindex attribute and value.
